HHS/OPRE-OPS
# fix: permission check for approve Agreement page

## What changed

- protect the Approve Agreement page from hackerz
- adds vscode settings for python tests
- adds roles for cypress e2e tests

## Issue

#2833 

## How to test

1. create some change requests
2. try and view the approve page logged in as admin
3. should fail
4. try and view the approve page logged in as division director
5. will work

## Screenshots

https://github.com/user-attachments/assets/7cf76d2c-6a3d-4a21-9a1a-c0c9f4bfee0e

## Definition of Done Checklist
